src/app/store/root.epic.ts

```diff
@@ -0,0 +1,10 @@
+import { combineEpics } from 'redux-observable';
+
+import { RootState, RootAction } from './root.reducer';
+import { clearProfile } from '../profiles/store/profile.epics';
+import { searchShows } from '../tvshows/store/tvshows.epics';
+
+export const rootEpic = combineEpics<RootAction, RootState>(
+  searchShows,
```

Comment:
Wouldn't the name of this key be `tvshows` to reflect the feature name?

Reply:
Likewise, this is another side effect function, which queries the TV database using HTTP.